src/VisualStudio/Core/Impl/Options/AbstractOptionPageControl.cs

```diff
@@ -177,4 +188,19 @@ public class RadioButtonCheckedConverter : IValueConverter
             return value.Equals(true) ? parameter : Binding.DoNothing;
         }
     }
+
+    public class CheckBoxCheckedToIntConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter,
+            System.Globalization.CultureInfo culture)
+        {
+            return !value.Equals(-1);
```

Comment:
nit: It probably doesn't matter at all, but why is this different from Convert.ToInt32? https://docs.microsoft.com/en-us/dotnet/api/system.convert.toint32?redirectedfrom=MSDN&view=netframework-4.7.2#System_Convert_ToInt32_System_Boolean_

Reply:
I will add a comment. This was to keep compatibility with the setting used in the Preview Features page. That value is an integer where -1 was disabled, 0 was use the flight information, and 1 was enabled.

Reply:
Ah, so this may not even be correct for all options? Yea, maybe a comment or rename is warranted here to make sure the intended use is understood